### DISCUSSION TITLE
Is it possible to add .html at the end of the URL as you like?

### DISCUSSION THREAD

**Discussion by gd-vanloi** [nuxt/nuxt#9449] [Questions (Nuxt 2)]
I am currently planning to develop a web application using NuxtJS framework with SSR Universal architecture, I am facing difficulties related to SEO, because previously my project was written in .NET , so all URLs have `.html` at the end..

Ex:
- domain.com/hotels (this is Cate)
- domain.com/hotels-5-star.html (this is a Sub cate of Hotels Cate)

The example above is specific to my problem, which means that according to Google, the shorter the URL, the better, so my pages have all been shortened to a single **"/"**, however because of the structure page has 2 hierarchies, **Cate and Sub cate**, so for Google to distinguish this, the URL (sub cate) must insert `.html` at the end.

Can I use NuxtJS build pages with the same URL as above?, I have researched but found a specific solution, maybe my experience is not much so I need the help of experts on NuxtJS.

Thank you very much experts!

**Comment by kissu** [comment]
Here is the question on SO: https://stackoverflow.com/q/68028690/8816585

You're trying to look for more info than the answer you've accepted?
